### DISCUSSION TITLE
Can't create again a deleted organization

### DISCUSSION THREAD

**Discussion by fabio-ivona** [community/community#33371] [Other Feature Feedback, Questions, & Ideas]
Hi all!

I have an issue with an organization that belonged to me, I deleted it a couple of days ago and wanted to create an organization with the same name, but when I try I receive this error message:  `Organization name 'defstudio' is unavailable.`

this happens both during a new organization creation and trying to rename an existing organization to that name

Anyone can hint me on why this happens?

ps: I'm not interested in the data, so no recovery, I just wish to re-use that name

**Comment by airtower-luna** [comment]
That sounds similar to the mechanism that [prevents popular repository names from being taken over](https://github.blog/2018-04-18-new-tools-for-open-source-maintainers/#popular-repository-namespace-retirement), but that post mentions account names aren't included. I wonder if they changed that since? :thinking: 

If you want to be sure it's probably best to [contact GitHub support](https://support.github.com/request).

**Reply by fabio-ivona** [reply]
Will do it, thanks!

**Comment by ishtiaqSamdani** [comment]
I am also facing same issue,
did you got any solution

**Reply by fabio-ivona** [reply]
Customer support just handled it, you only have to open a ticket if the previous organization belonged to you

otherwise you have to wait 90 days in order to the name to be released

**Comment by Jiya-Ull-Haq** [comment]
I'm facing the same issue.
Let me know if you've got any solution.
Thanks

**Reply by ishtiaqSamdani** [reply]
looks like we found a bug 🐛

**Reply by fabio-ivona** [reply]
Customer support just handled it, you only have to open a ticket if the previous organization belonged to you

otherwise you have to wait 90 days in order to the name to be released

**Reply by Jiya-Ull-Haq** [reply]
Thanks :)
